ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -467,6 +467,18 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: aws-ipi-p3-f28-long-duration-apiserver
```

Comment:
p3 is lower, p2 is better, [Priority Definition](https://docs.google.com/document/d/1-ODg26Ho2AUKCJ-7kpJizmCrUunjAHteWvlC4yG1PxY/edit#heading=h.2vpiramvcdo1), tested at least once.
f28 means every 28 days, for z-streams, we need to run the case at least once a week. We had better run the test every Friday, check the results on next Monday, f7 is fine.
Cron job: `29 18 * * 5`, run test at 18:29 every Friday.

Reply:
@wangke19 Updated the suggested changes.